src/base/bittorrent/session.cpp

```diff
@@ -2353,9 +2353,9 @@ bool Session::addTorrent_impl(const std::variant<MagnetUri, TorrentInfo> &source
     p.added_time = std::time(nullptr);
 
     if (!isFindingIncompleteFiles)
-        return loadTorrent(loadTorrentParams);
+        return loadTorrent(std::move(loadTorrentParams));
 
-    m_loadingTorrents.insert(id, loadTorrentParams);
+    m_loadingTorrents.emplace(id, std::move(loadTorrentParams));
```

Comment:
as said before, qt containers only supports copy-semantics, so move-semantics won't work.

Reply:
>Seems Qt6 changed it.

With the latest push I fixed this so that it works with Qt5 as well (of course, not as efficient as `emplace()`, since it needs first to create a default object and then move there).